text/0000-remove-traits.md

```diff
@@ -0,0 +1,71 @@
+- Feature Name: Remove Traits from Pony
+- Start Date: 2019-01-11
+- RFC PR: 
+- Pony Issue: 
+
+# Summary
+
+Remove traits from Pony, leaving only interfaces.
+
+# Motivation
+
+As it currently stands with Pony. Traits and Interfaces are almost identical.  There's nothing you can do with one that you can't do with the other.
+
+New users coming to Pony often end up in IRC asking what the difference between the two are. We generally explain that traits are nominal subtyping and interfaces are structural subtyping and this is mostly true except, that you use the `is` keyword when implementing an interface to get compile time checks to make sure you are implementing said interface.
+
+All in all, its rather confusing. 
+
+Where possible, we should be aiming to provide the minimum number of concepts in Pony to allow a user to solve the problem they have at hand. There is no problem that traits solves that can not be solved with interfaces. There are however things you can do with structual subtyping and interfaces that you can't do with traits.
```

Comment:
For completeness of the RFC, can we mention a couple of examples attributing to the fact that "there are however things you can do with interfaces that one can't with traits" ?

Reply:
@blitzerr  i've expanded upon that.